### ISSUE TITLE
Roundtrip on LocalDateTime doesn’t work

### ISSUE THREAD

**Issue by gurghet** [sksamuel/avro4s#375]
I will open a pull request on this if nobody does. Anyways I wanted to create an issue to keep track.

```
val localDateTime = LocalDateTime.of(2018, 1, 1, 23, 30, 5, 328187943)
val encodedLocalDateTime = Encoder[LocalDateTime].encode(
  localDateTime,
  AvroSchema[LocalDateTime],
  DefaultFieldMapper
)
Decoder[LocalDateTime]
  .decode(encodedLocalDateTime, AvroSchema[LocalDateTime], DefaultFieldMapper) shouldEqual localDateTime
```
```
Expected :2018-01-01T23:30:05.328187943
Actual   :2018-01-01T23:30:05.328
```

**Comment by sksamuel** [comment]
Please do raise a PR thanks.

**Comment by sksamuel** [comment]
I've fixed this and released 3.0.3.
